packages/sith-language-server/package.yaml

```
@@ -0,0 +1,44 @@
+---
+name: sith-language-server
+description: An experimental language server for the Python programming language. - made in Rust.
+homepage: https://github.com/LaBatata101/sith-language-server
+licenses:
+  - MIT
+languages:
+  - Python
+categories:
+  - LSP
+
+source:
+  id: pkg:github/LaBatata101/sith-language-server@v0.2.2-alpha
+  asset:
+    - target: darwin_arm64
+      file: sith-lsp-aarch64-apple-darwin.tar.gz
+      bin: sith-lsp-aarch64-apple-darwin/sith-lsp
+    - target: darwin_x64
+      file: sith-lsp-x86_64-apple-darwin.tar.gz
+      bin: sith-lsp-x86_64-apple-darwin/sith-lsp
+    - target: linux_arm64_gnu
+      file: sith-lsp-aarch64-unknown-linux-gnu.tar.gz
+      bin: sith-lsp-aarch64-unknown-linux-gnu/sith-lsp
+    - target: linux_arm64_musl
+      file: sith-lsp-aarch64-unknown-linux-musl.tar.gz
+      bin: sith-lsp-aarch64-unknown-linux-musl/sith-lsp
+    - target: linux_x64_gnu
+      file: sith-lsp-x86_64-unknown-linux-gnu.tar.gz
+      bin: sith-lsp-x86_64-unknown-linux-gnu/sith-lsp
+    - target: linux_x64_musl
+      file: sith-lsp-x86_64-unknown-linux-musl.tar.gz
+      bin: sith-lsp-x86_64-unknown-linux-musl/sith-lsp
+    - target: win_x64
+      file: sith-lsp-x86_64-pc-windows-msvc.zip
+      bin: sith-lsp-x86_64-pc-windows-msvc/sith-lsp.exe
+    - target: win_arm64
+      file: sith-lsp-aarch64-pc-windows-msvc.zip
+      bin: sith-lsp-aarch64-pc-windows-msvc/sith-lsp.exe
+
+schemas:
+  lsp: vscode:https://raw.githubusercontent.com/LaBatata101/sith-language-server/main/editors/vscode/package.json
```

```suggestion
  lsp: vscode:https://raw.githubusercontent.com/LaBatata101/sith-language-server/{{version}}/editors/vscode/package.json
```

we should get the correct schema given the current version